.github/workflows/dispatch-sync-to-private.yml

```diff
@@ -0,0 +1,19 @@
+name: Dispatch sync event to plasma-contracts-private
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  dispatch-sync-event:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Repository Dispatch
+        if: github.repository == 'omisego/plasma-contracts'
+        uses: peter-evans/repository-dispatch@v1
```

Comment:
Who is peter evans? Sorry. I haven’t used gh actions before.

Reply:
I don't know either. I just randomly picked from the Github action market place
https://github.com/marketplace/actions/repository-dispatch

Reply:
okay, since it is fairly simple, I tried to remove those marketplace dependencies with shell scripts instead.